examples/box2d/bipedal_hardcore_sac.py

```diff
@@ -60,13 +60,22 @@ def __init__(self, env, action_repeat=3, reward_scale=5, rm_done=True):
     def step(self, action):
         rew_sum = 0.0
         for _ in range(self.action_repeat):
-            obs, rew, done, info = self.env.step(action)
+            step_result = self.env.step(action)
+            if len(step_result) == 4:
```

Comment:
This should never happen, we use gymnasium envs, not (older) gym envs. How did you run into this?

Reply:
Note that if it needs fixing in one example, it will need fixing in all of them :(

Reply:
I guess it was some older example that noone touched in a while. The check for old vs new api is still present in atari_wrapper.py, that's where I just copied it from. You're right though, the Collector assumes terminated and truncated coming from env.step, I'll just delete this part